alcs-frontend/src/app/features/board/board.component.scss

```diff
@@ -4,12 +4,16 @@
   display: flex;
   flex-direction: column;
   height: 100%;
+  overflow: auto;
 }
 
 .board-header {
   display: flex;
   justify-content: space-between;
+  align-items: center;
   padding: 24px 80px 12px 76px;
+  position: sticky;
```

Comment:
If you are setting sticky here, is it still needed on .column-header?

Reply:
Just tested it and seems like we can remove it from `.column-header`, I'll go ahead and clean that up